composer.json

```
@@ -29,7 +29,7 @@
         "source": "https://github.com/browscap/browscap-php"
     },
     "require": {
-        "php": ">=7.4.3,<8.3.0",
+        "php": ">=7.4.3,^8.3.0",
```

Since PHP is not semver, using the "semver" operator is incorrect.

```suggestion
        "php": ">=7.4.3,<8.4.0",
```

Please could you also expand the CI matrix?

---

Done. I don't have workflows setup, but it should run fine.

---

I didn't update the individual tools though. I saw two had PHP81 so not sure if that was done on purpose or not.